cegui/src/RenderedStringWidgetComponent.cpp

```diff
@@ -101,19 +101,12 @@ std::vector<GeometryBuffer*> RenderedStringWidgetComponent::createRenderGeometry
     std::vector<GeometryBuffer*> geomBuffers;
 
     if (!window)
-        std::vector<GeometryBuffer*>();
```

Comment:
ouch, nice find

Reply:
Static analysis from VS2019. I've run it through the whole codebase and fixed the most of issues. This was one of the most dangerous.